MIMWebClient/Core/ItemContainer.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace MIMWebClient.Core
+{
+
+    public class ItemContainer : List<Item.Item>
+    {
+        public IEnumerable<String> List()
+        {
+            return List(this, false);
+        }
+
+        public IEnumerable<String> List(bool article)
+        {
+            return List(this, article);
+        }
+
+        public static IEnumerable<String> List(IEnumerable<Item.Item> items)
+        {
+            return List(items, false);
+        }
+
+        public static IEnumerable<String> List(IEnumerable<Item.Item> items, bool article)
+        {
+            return items.GroupBy(t => t.name).Select(group => new
+            {
+                Name = group.Key,
+                Count = group.Count()
+            }).Select(x =>
+            {
+                var itemString = x.Name + ((x.Count > 1) ? " x" + x.Count : "");
+                if (article)
+                {
+                    var result = AvsAnLib.AvsAn.Query(x.Name);
+                    itemString = result.Article + " " + itemString;

```

Comment:
result.Article should start with a capital letter. I have a helper method here: [Helpers.FirstLetterToUpper(string)](https://github.com/LiamKenneth/ArchaicQuest/blob/b46dd24133fca8529f4c07e79b138848cbd4f896/MIMWebClient/Core/Helpers.cs#L178)